pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -229,7 +229,7 @@ func init() {
 	addControllerRole(rbac.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + InfraClusterQuotaReconciliationControllerServiceAccountName},
 		Rules: []rbac.PolicyRule{
-			rbac.NewRule("get", "list").Groups(kapiGroup).Resources("configMaps").RuleOrDie(),
+			rbac.NewRule("get", "list").Groups(kapiGroup).Resources("configmaps").RuleOrDie(),
```

Comment:
Add a linting test on the bootstrap ones that requires lowercase.  I'm not sure enough to make `RuleOrDie` fail, but a linting test could have a whitelist.